src/DayTime.cpp

```diff
@@ -69,20 +69,31 @@ DayTime::DayTime(const DayTime &other)
 
 DayTime::DayTime(int h, int m, int s)
 {
-  LOGV4(DEBUG_INFO, "DayTime ctor(%d, %d, %d)", h, m, s);
+  //LOGV4(DEBUG_INFO, "DayTime ctor(%d, %d, %d)", h, m, s);
   long sgn = sign(h);
   h = abs(h);
   totalSeconds = sgn * ((60L* h + m) * 60L + s);
-  LOGV6(DEBUG_INFO, "DayTime ctor(%d, %d, %d) -> %l -> %s", h, m, s, totalSeconds, ToString());
+  //LOGV6(DEBUG_INFO, "DayTime ctor(%d, %d, %d) -> %l -> %s", h, m, s, totalSeconds, ToString());
 }
 
 DayTime::DayTime(float timeInHours)
 {
-  LOGV2(DEBUG_INFO, "DayTime ctor(%f)", timeInHours);
+  //LOGV2(DEBUG_INFO, "DayTime ctor(%f)", timeInHours);
   long sgn = fsign(timeInHours);
   timeInHours = abs(timeInHours);
   totalSeconds = sgn * round(timeInHours * 60 * 60);
-  LOGV4(DEBUG_INFO, "DayTime ctor(%f) -> %l -> %s", timeInHours, totalSeconds, ToString());
+  //LOGV4(DEBUG_INFO, "DayTime ctor(%f) -> %l -> %s", timeInHours, totalSeconds, ToString());
+}
+
+DayTime& DayTime::operator=(DayTime const& other) // copy assignment
```

Comment:
Agreed. I will review the existing implementation and fix this.

Reply:
Actually, we can remove hourWrap from this class, since I derived the Declination class (needs a different wrap value). The wrapping is now handled in the virtual checkHours() method.

Reply:
I've added hourWrap to the copy constructor - good catch! The other constructors initialize from declaration.
@ClutchplateDude Can you please take a look at hourWrap? DayTime::checkHours() is still using it.

Reply:
Just to verify, the hourWrap was eliminated, right?

Reply:
nvm, I saw the commit...